src/pages/organizers/create/OrganizerForm.tsx

```diff
@@ -143,8 +181,10 @@ const OrganizerForm = () => {
 
   const onSuccess = () => {
     upsertOrganizer({
-      onSuccess: async (organizerId) =>
-        await push(`/organizers/${organizerId}/edit`),
+      onSuccess: async (organizerId: string) => {
+        handleCultuurkuurLabelMutation(organizerId);
```

Comment:
Think you would need to await this too otherwise the label change won't be reflected yet when the page changes